### DISCUSSION TITLE
iPadでは、インライン数式の前で勝手に改行が入る

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#425] [要望・アイデア・不具合]
これは、数式表示の不具合の改良（2022-11-15付、無駄なスクロールバーの排除など）以前からあった問題です。

　次のスクリーンショットは私が投稿した記事の一部で、上がiPadの画面、下がWindowsの画面です。iPadのインライン数式の前には、不本意な改行がたくさん入ってきています。

<img width="706" alt="iPadで表示Qiita" src="https://user-images.githubusercontent.com/42261605/215305791-c6210577-be30-4745-831c-3d59f02d8903.png">
<img width="553" alt="Windowsで表示Qiita" src="https://user-images.githubusercontent.com/42261605/215305797-dcb1b61c-8752-4e5d-8f5c-948b649f9100.png">

　ただ、私はWindowsメインで使用しているし、iOS遣いの方からは何のクレームも出ていないようなので、今までは特に言及したことはありませんでした。

　しかし、最近、下記記事に、これを指しているようなコメントを見つけましたので、遅ればせながら、勝手に代理でDiscussionさせていただきました。

> スミルノフ・グラブス検定
> https://qiita.com/xxyurii/items/3dd9d41dd96b4103e82c

これへの編集リクエストに対して、投稿主のコメントは、

> 文中の数式を$$で囲んでないことについては、macOSのsafariにおいては、一つの段落に二つ以上の数式を置くと表示が乱れる現象があるためです。どのような対応にすべきか迷っています。

しかし、投稿主はリクエストに折れて改訂してしまっているようで、Windowsでは正しく表示されているものの、iPadでは残念な表示になっています。

　これはQiitaのシステム改善でどうにかできる問題なのか？　AppleのOSのどうしようもない癖なのか？　投稿時の裏技でなんとかできるのか？　全く分かりませんが、可能なら改善いただけると嬉しいです。折角投稿した記事なので、どのような環境の方にも筆者の意図どおりに読んでいただけるのが理想です。

**Comment by tanaka-asami415** [comment]
@tsubolabo 
ご報告ありがとうございます🙏
数式関連のご意見は度々いただいているものの、対応できておらずすみません 🙇
今回頂いたものも合わせて、引き続き調査いたします。何卒ご了承ください 。

**Comment by kyntk** [comment]
@tsubolabo
https://github.com/increments/qiita-discussions/discussions/498 でもお伝えしましたが、MathJaxのバージョンアップを行いました。

上記ご報告いただいた内容については、このバージョンアップで解消される見込みなのですが、何か変化はあるでしょうか？
お手すきの際にご確認いただけると幸いです🙏

**Reply by tsubolabo** [reply]
@kyntk 

今回のV3化により、iPadにおけるインライン数式前の強制改行だけは改善されました。ありがとうございました。

しかし、今回のV3化は、私には副作用の方が大きく感じられます。#298は全く改善されていないうえに、#498で指摘されているベースラインの新たなズレまで加わりとても許容できません。しかし、#498に対する貴コメントを拝見すると、すぐに直せるという雰囲気が感じられません。

このまま長期間、数式が崩れた投稿を見ると思うと気分が滅入ります。また新しい記事への創作意欲も削がれます。できることなら、直ぐにでもV2に戻し、V3は水面下でもっと十分に作り込んでから再デビューさせることにしてはいかがでしょうか？

**Reply by kyntk** [reply]
@tsubolabo ご確認いただきありがとうございます :bow: 
解消されているようでよかったです。

#498 でも記載したのですが、今回のアップデートでは様々な改善やバグ修正がされています。
プラス面も鑑みて、今後検討させていただきます :pray:
